packages/lib-classifier/src/store/SubjectStore.spec.js

```diff
@@ -51,97 +46,78 @@ describe('Model > SubjectStore', function () {
 
     describe('with less than three subjects in the queue', function () {
       let populateSpy
+      let rootStore
 
       before(function () {
+        rootStore = setupStores()
         populateSpy = sinon.spy(rootStore.subjects, 'populateQueue')
-        while (rootStore.subjects.resources.size > 2) {
-          rootStore.subjects.advance()
-        }
       })
 
       after(function () {
         rootStore.subjects.populateQueue.restore()
+        rootStore = null
       })
 
       it('should request more subjects', function () {
+        while (rootStore.subjects.resources.size > 2) {
+          rootStore.subjects.advance()
+        }
         expect(populateSpy).to.have.been.calledOnce()
       })
     })
 
     describe('after emptying the queue', function () {
+      let rootStore
       before(function () {
         sinon.stub(clientStub.panoptes, 'get').callsFake(() => Promise.resolve({ body: [] }))
-        while (rootStore.subjects.resources.size > 0) {
-          rootStore.subjects.advance()
-        }
+        rootStore = setupStores()
       })
 
       after(function () {
         clientStub.panoptes.get.restore()
+        rootStore = null
```

Comment:
This isn't needed. `rootStore` is scoped to this function with `let`, so it will be destroyed when the describe block finishes.